cpp/models/ode_seir/model.h

```diff
@@ -101,36 +103,39 @@ class Model : public CompartmentalModel<InfectionState, Populations<InfectionSta
         auto num_time_points = y.get_num_time_points();
         Eigen::VectorXd temp(num_time_points);
         for (int i = 0; i < num_time_points; i++) {
-            temp[i] = get_reproduction_number(i, y).value();
+            temp[i] = get_reproduction_number(static_cast<size_t>(i), y).value();
         }
         return temp;
     }
 
     /**
-    * @brief interpolate reproduction numbers at freely chosen time value that lies in between the time points of the given time series.
-    * values are linearly interpolated from their immediate neighbors from the old time points.
-    * @param y TimeSeries whose reproduction numbers we want to interpolate
-    * @param t_value double time at which we approximate the reproduction number
-    * @return interpolated reproduction number at given time value
+    *@brief Computes the reproduction number at a given time point of the Model output obtained by the Simulation. If the particular time point is not inside the output, a linearly interpolated value is returned.
```

Comment:
This is probably not formatted?

Reply:
I have clang autoformat working. So it should be formatted correctly